src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -299,7 +304,7 @@ public ChatView(ChatModel chat)
             _ChatModel = chat;
 
             IsAutoScrolling = true;
-            MessageTextView tv = new MessageTextView();
+            MessageTextView tv = CreateMainMessageTextView();
```

Comment:
I like "var", but only when without it there would be repetition of the type being written. If it's a call to a method, it's better to not use 'var', so you can see the type there in the same line without looking at the method's tooltip/implementation.